lib/src/main/java/com/auth0/android/lock/Lock.java

```diff
@@ -436,6 +436,24 @@ public Builder setDefaultDatabaseConnection(String connectionName) {
             return this;
         }
 
+        /**
+         * Set which enterprise connections should use Web Authentication instead of the Username
+         * and Password form.
+         * Enterprise connections based on 'ad', 'adfs' and 'waad' strategies can log their
+         * users in from within the Lock widget using their email and password. This is known as
+         * Active Authentication, and is not supported for MFA enabled identity providers.
+         * By whitelisting the connections here, the Universal Login Page is used instead and the
+         * log in is delegated to the browser application.
```

Comment:
I don't know to be honest. I know about MFA because it was reported now. Should I remove it?

Reply:
This whole paragraph just seems like overkill. Maybe just:

> Set which enterprise connections should use Web Authentication (redirect to the Universal Login Page) instead of Active Authentication (username and password form).

Reply:
I removed that and the MFA mention.